build.gradle

```diff
@@ -80,4 +82,28 @@ githook {
 
 commitlint {
     enforceRefs = false
-}
\ No newline at end of file
+}
+
+openApiGenerate {
+  generatorName = 'spring'
+  inputSpec = "$rootDir/frontend/openapi.yaml"
+  outputDir = "$buildDir/generated"
+  apiPackage = 'com.ssegning.lynx.lynxbackend.api'
+  modelPackage = 'com.ssegning.lynx.lynxbackend.model'
+  invokerPackage = 'com.ssegning.lynx.lynxbackend.handler'
+  
+  configOptions = [
+    useTags: 'true',
+    library: 'spring-boot',
+    serializableModel: 'true',
+    dateLibrary: 'java8',
+    useBeanValidation: 'true',
+    performBeanValidation: 'true',
+    interfaceOnly: 'true',
+    useOptional: 'false',
+    useSpringBoot3: 'true',
+    useResponseEntity: 'false'
+  ]
+}
+
+tasks.named("openApiGenerate") { dependsOn("build") }
```

Comment:
Good but the opposite. It's the build that should depends on the openapi generation.